src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapOasis.java

```diff
@@ -32,6 +34,13 @@ public void vuapDefaultStateShouldStartPlayingAdvertisementAutomatically(Page pa
     vuap.waitForVideoStart();
 
     VuapAssertions.verifyVideoPlay(vuap);
+
+    PageObjectLogging.log(
+        "Take screenshot",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Take screenshot" 14 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

Reply:
Can we extract it to some private method? (if not annotation)